pkg/winutil/io.go

```diff
@@ -0,0 +1,26 @@
+package winutil
```

Comment:
do we need to compile this for non-windows?

Reply:
Not really, but it doesn't hurt I guess? I can add a build directive if you want

Reply:
I guess it's fine this way since this would not throw any compilation errors on darwin as well.